src/btree/bt_delete.c

```diff
@@ -237,13 +237,13 @@ __wt_delete_page_skip(WT_SESSION_IMPL *session, WT_REF *ref, bool visible_all)
      * being read into memory right now, though, and the page could switch to an in-memory state at
      * any time. Lock down the structure, just to be safe.
      */
-    if (ref->page_del == NULL && ref->page_las == NULL)
+    if (ref->page_del == NULL && !ref->has_las)
```

Comment:
Do we still need the `ref->has_las` member? Is this member is used to find out whether this page has any history store records? If yes, once this flag is set, it is never be set to false again in normal scenarios.